src/components/Skeleton/PseudoText.tsx

```diff
@@ -7,7 +7,7 @@ import { ILoaderInfo, IPseudoText } from './types';
 import { normalizeSize } from './utils';
 
 export const PseudoText = forwardRef<HTMLSpanElement, IPseudoText>(
-  ({ width: propsWidth = `${getRandomInt(20, 60)}%`, height: propsHeight, className }, ref) => {
+  ({ width: propsWidth = `${getRandomInt(250, 350)}px`, height: propsHeight, className }, ref) => {
```

Comment:
this change will break every element using PseudoText without width property. Percentages should be used for PseudoText to be abstract and do not depend on actual usage place